### ISSUE TITLE
[告警通知优化] <告警通知存在辨识度很差，建议稍微优化一下语言>

### ISSUE THREAD

**Issue by macaty** [dromara/hertzbeat#935]
### Description

目前的告警通知存在辨识度很低，导致要花很多经历观察告警，建议通过简单的醒目的icon、颜色来标识，一眼就能发现问题
1、目前告警样例

**Comment by hertzbeat** [comment]
> Bot detected the issue body's language is not English, translate it automatically. 👯👭🏻🧑‍🤝‍🧑👫🧑🏿‍🤝‍🧑🏻👩🏾‍🤝‍👨🏿👬🏿         
----    
### Description

The current alarm notification has a low degree of recognition, which requires a lot of experience to observe the alarm. It is recommended to use a simple eye-catching icon and color to identify the problem, and you can find the problem at a glance.
1. Current alarm example

### Task List

_No response_

**Comment by Neilblaze** [comment]
@hertzbeat I'm a newbie & I want to take a shot at this. I'm pretty good at English, & for Mandarin, I use Google Translate. Would you please help me out with this? Thanks!

**Comment by tomsun28** [comment]
hi @Neilblaze, the @hertzbeat is a robot. Do you want to develop and implement this issue feature task ? very welcome!

**Comment by Neilblaze** [comment]
Yep, I'll be down to contribute! Also, would you mind taking a peek at the gitter? Thanks! 😄  
> P.S. > I tagged you in my message.

**Comment by Neilblaze** [comment]
@tomsun28 Yep, I'll be down to contribute! Also, would you mind taking a peek at the gitter? Thanks! 😄  
> P.S. > I tagged you in my message.

**Comment by tomsun28** [comment]
ok 👍
